restapi/configure_qdb_api_rest.go

```diff
@@ -218,12 +220,20 @@ func configureAPI(api *operations.QdbAPIRestAPI) http.Handler {
 	APIConfig.SetDefaults()
 
 	if APIConfig.Log != "" {
-		f, err := os.OpenFile(string(APIConfig.Log), os.O_RDWR|os.O_CREATE|os.O_APPEND, 0644)
+		_, err := os.OpenFile(string(APIConfig.Log), os.O_RDWR|os.O_CREATE|os.O_APPEND, 0644)
```

Comment:
I may be missing something, but why is it necessary to open this file and/or where is it closed? It strikes me as if lumberjack itself takes care of the opening/closing of the files, right?

Reply:
Intention was-> just to check whether target folder for log is writable. You are right, we should not let this FD remain unnamed.

Reply:
added defer call to close file